main.js

```diff
@@ -1,51 +1,63 @@
+const submitButton = document.getElementById('submit');
+let bookCollection = [];
+const booksContainer = document.getElementById('books-container');
+
+/* BOOK CLASS */
 class Book {
   constructor(title, author) {
     this.title = title;
     this.author = author;
   }
-}
 
-let bookCollection = [];
-
-const submitButton = document.getElementById('submit');
-const booksContainer = document.getElementById('books-container');
+  static addButton(e) {
+    e.preventDefault();
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
 
-const addButton = (e) => {
-  e.preventDefault();
-  const title = document.getElementById('title').value;
-  const author = document.getElementById('author').value;
+    const newBook = new Book(title, author);
 
-  // add new book to bookCollection
-  const newBook = new Book(title, author);
-  bookCollection.push(newBook);
+    bookCollection.push(newBook);
 
-  // add book object to local storage
-  localStorage.setItem('books', JSON.stringify(bookCollection));
```

Comment:
- [ ] The requirement says that the previous functionality should be maintained. The previous milestone included using local storage to save data it is saving fine but when I refresh pages it disappeared So kindly make changes to it.